src/lib.rs

```diff
@@ -159,119 +157,111 @@ To see all the flags the proxied tool accepts run `cargo-{} -- --help`.{}",
+            Arg::new("jobs")
                 .long("jobs")
-                .short("j")
+                .short('j')
                 .value_name("N")
                 .help("Number of parallel jobs, defaults to # of CPUs"),
-            Arg::with_name("lib")
+            Arg::new("lib")
                 .long("lib")
-                .conflicts_with_all(&["bin", "example", "test", "bench"])
+                .conflicts_with_all(["bin", "example", "test", "bench"])
                 .help("Build only this package's library"),
-            Arg::with_name("bin")
+            Arg::new("bin")
                 .long("bin")
-                .takes_value(true)
+                .action(ArgAction::Set)
                 .value_name("NAME")
-                .conflicts_with_all(&["lib", "example", "test", "bench"])
+                .conflicts_with_all(["lib", "example", "test", "bench"])
                 .help("Build only the specified binary"),
-            Arg::with_name("example")
+            Arg::new("example")
                 .long("example")
-                .takes_value(true)
+                .action(ArgAction::Set)
                 .value_name("NAME")
-                .conflicts_with_all(&["lib", "bin", "test", "bench"])
+                .conflicts_with_all(["lib", "bin", "test", "bench"])
                 .help("Build only the specified example"),
-            Arg::with_name("test")
+            Arg::new("test")
                 .long("test")
-                .takes_value(true)
+                .action(ArgAction::Set)
                 .value_name("NAME")
-                .conflicts_with_all(&["lib", "bin", "example", "bench"])
+                .conflicts_with_all(["lib", "bin", "example", "bench"])
                 .help("Build only the specified test target"),
-            Arg::with_name("bench")
+            Arg::new("bench")
                 .long("bench")
-                .takes_value(true)
+                .action(ArgAction::Set)
                 .value_name("NAME")
-                .conflicts_with_all(&["lib", "bin", "example", "test"])
+                .conflicts_with_all(["lib", "bin", "example", "test"])
                 .help("Build only the specified bench target"),
-            Arg::with_name("release")
+            Arg::new("release")
                 .long("release")
                 .help("Build artifacts in release mode, with optimizations"),
-            Arg::with_name("profile")
+            Arg::new("profile")
                 .long("profile")
                 .value_name("PROFILE-NAME")
                 .help("Build artifacts with the specified profile"),
-            Arg::with_name("features")
+            Arg::new("features")
                 .long("features")
-                .multiple(true)
-                .number_of_values(1)
-                .takes_value(true)
+                .action(ArgAction::Set)
                 .value_name("FEATURES")
                 .help("Space-separated list of features to activate"),
```

Comment:
im not super confident about this, why did we specify multiple here?

anyway, this is wrong, should be `ArgAction::Append`